src/genie/libs/parser/iosxe/show_vrrp.py

```diff
@@ -158,15 +158,16 @@ def cli(self, interface='', group='', output=None):
             r'^(?P<interface>[\w,\.\/]+)\s+-\s+Group\s(?P<group_number>\d+)(\s+-\s+Address-Family\s+(?P<address_family>IPv4|IPv6))?$')
 
         #State is Master
-        p2 = re.compile(r'State is (?P<state>(Master|MASTER|Up|UP|Init|INIT))$')
+        p2 = re.compile(r'State is (?P<state>(Master|MASTER|Up|UP|Init|INIT)).*$')
```

Comment:
Please comment the line this is parsing above this

Reply:
Made the changes as suggested